code/campaign/src/main/java/com/adobe/marketing/mobile/campaign/CampaignExtension.java

```diff
@@ -872,16 +875,33 @@ private void migrateFromACPCampaign(final NamedCollection aepDatastore) {
         }
 
         SharedPreferences sharedPreferences = null;
-        final Context appContext = MobileCore.getApplication().getApplicationContext();
```

Comment:
nit: getApplication is expected to be non null because the Core starts extension registration only after the application instance has been set by the app.